.circleci/config.yml

```diff
@@ -203,7 +203,7 @@ jobs:
           command: |
             docker run -ti --rm=false \
               --entrypoint="/usr/local/miniconda/bin/py.test" \
-              poldracklab/fmriprep:latest . \
+              poldracklab/fmriprep:latest /root/src/fmriprep/ \
```

Comment:
Is it possible to just run `py.test fmriprep`? Since it's installed, it should test the installed version, not the source that happens to be included in the image. While tests are rarely run by non-developers, I'd hope they'd work equally well when installed.